### PR TITLE
updated entity usage link due to new column names in v2 release

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Updated columns names with the update in entity usage.

# Needed By (Date)
- asap

# Urgency
- low

# Steps to Test

1. use the branch and update composer
2. view the media usage page at `/admin/content/media/{mid}`
3. verify no errors occur.

# Affected Projects or Products
- MRC
- Arch

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/install/views.view.media_entity_browser.yml [hunk 1]
```diff
@@ -414,10 +414,15 @@ display:
         - user.permissions
       tags:
         - 'config:core.entity_view_display.media.file.default'
+        - extensions
```
Comment (sherakama) [inline config/install/views.view.media_entity_browser.yml]: Do we need extensions this many times in the array?

File: config/install/views.view.media_entity_browser.yml [hunk 2]
```diff
@@ -1708,10 +1730,15 @@ display:
...
```
Comment (sherakama) [inline config/install/views.view.media_entity_browser.yml]: Do we need extensions this many times in the array?

File: config/install/views.view.media_entity_browser.yml [hunk 3]
```diff
@@ -1834,7 +1861,12 @@ display:
...
```
Comment (sherakama) [inline config/install/views.view.media_entity_browser.yml]: Do we need extensions this many times in the array?

**Comment by pookmish** [comment]
I refershed the configs from a recent `csex` and those `- extensions` are gone. not sure where they even came from.

File: config/install/embed.button.media_browser.yml [hunk 4]
```diff
@@ -15,4 +15,4 @@ type_settings:
   entity_browser: media_browser
   entity_browser_settings:
     display_review: false
-icon_uuid: null
+icon_uuid: 3ed72229-7529-4cf5-b679-1d7a86b1030a
```
Comment (sherakama) [inline config/install/embed.button.media_browser.yml]: Is this a reference to a file?

Comment (sherakama) [reply config/install/embed.button.media_browser.yml]: Ah, I see it is and you have hardwired it in the module. Ok. Ignore this comment.

Comment (pookmish) [reply config/install/embed.button.media_browser.yml]: yeah. it just seemed easier since each site couldve potentially had different file uuids.

**Pull request merged**